lib/connection.cpp

```diff
@@ -2497,14 +2497,14 @@ void Connection::sendToDevice(const QString& targetUserId,
 bool Connection::isVerifiedSession(const QString& megolmSessionId) const
 {
     auto query = database()->prepareQuery("SELECT olmSessionId FROM inbound_megolm_sessions WHERE sessionId=:sessionId;"_ls);
-    query.bindValue(":sessionId", megolmSessionId);
+    query.bindValue(":sessionId", megolmSessionId.toLatin1());
```

Comment:
These queries just don't belong here in the first place, tbh, they should be in `Database`; and to the point of `QString/QByteArray`, I'd probably just pass the session object itself here.

Reply:
Passing the session object itself wouldn't be ideal since this function is intended for clients to check if a session is verified and clients can get the sessionId relatively easily from the encryptedevent while getting the session object is more involved